dotCMS/src/main/java/com/dotcms/cache/VanityUrlCacheImpl.java

```diff
@@ -0,0 +1,120 @@
+package com.dotcms.cache;
+
+import com.dotcms.util.VanityUrlUtil;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.util.VanityUrlUtil'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)